docs/get_started/get_started_raspbian.md

```diff
@@ -0,0 +1,111 @@
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`share/`        | CMake configuration files for linking with Inference Engine.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`src/`          | Source files for CPU extensions.|
+
+</details>
+
+## <a name="workflow-overview"></a>OpenVINO™ Workflow Overview
+
+The OpenVINO™ workflow on Raspbian* OS is as follows:
+1. **Get a pre-trained model** for your inference task. If you want to use your model for inference, the model must be converted to the `.bin` and `.xml` Intermediate Representation (IR) files, which are used as input by Inference Engine. On Raspberry PI, OpenVINO™ toolkit includes only the Inference Engine module. The Model Optimizer is not supported on this platform. To get the optimized models you can use one of the following options:
+   
+   * Download public and Intel's pre-trained models from the [Open Model Zoo](https://github.com/opencv/open_model_zoo) using [Model Downloader tool](@ref omz_tools_downloader_README#model_downloader_usage).
+    <br> For more information on pre-trained models, see [Pre-Trained Models Documentation](@ref omz_models_intel_index)
+   
+   * Convert a model using the Model Optimizer from a full installation of Intel® Distribution of OpenVINO™ toolkit on one of the supported platforms. Installation instructions are available:
+     * [Installation Guide for macOS*](../install_guides/installing-openvino-macos.md)
+     * [Installation Guide for Windows*](../install_guides/installing-openvino-windows.md)
+     * [Installation Guide for Linux*](../install_guides/installing-openvino-linux.md)  
+2. **Use the Inference Engine API in the application** to run inference against the Intermediate Representation (optimized model) and output inference results. The application can be an OpenVINO™ sample or your own application. 
+
+## <a name="using-sample"></a>Build and Run Code Samples
+
+Follow the steps below to run pre-trained Face Detection network using Inference Engine samples from the OpenVINO toolkit.
+
+1. Create a samples build directory. This example uses a directory named `build`:
+```sh
+mkdir build && cd build
+```
+2. Build the Object Detection Sample with the following command:
+```sh
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino/deployment_tools/inference_engine/samples/cpp
+```
+```sh
+make -j2 object_detection_sample_ssd
+```
+3. Download the pre-trained Face Detection model with the Model Downloader:
+
+   - To download the `.bin` file with weights:
+   ```sh
+   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
+   cd open_model_zoo/tools/downloader
+   python3 -m pip install -r requirements.in
+   python3 downloader.py --name face-detection-adas-0001 
+   ```
+
+4. Run the sample, specifying the model and path to the input image:
+```sh
+./armv7l/Release/object_detection_sample_ssd -m face-detection-adas-0001.xml -d MYRIAD -i <path_to_image>
```

Comment:
Same steps are described at https://github.com/openvinotoolkit/openvino/blob/master/docs/install_guides/installing-openvino-raspbian.md#build-and-run-object-detection-sample

Reply:
@dkurt Installation Guide will be updated by 2021.2: these instructions will be removed, link to GSG will be added